src/app/PCM/components/pcm-field/non-pcm-field.tsx

```diff
@@ -38,20 +41,34 @@ const NonPcmPcmField: React.FC<NonPcmPcmFieldProps> = (props): JSX.Element => {
           question: ask.question,
           props: config,
         })}
-      {!isEvt && !child && (
-        <Attribute
-          key={code}
-          attribute={code}
-          code={config?.parentCode || ask?.targetCode}
-          size={config?.size}
-          mini={config?.mini}
-          parentCode={config?.parentCode || ask?.targetCode}
-          variant={config?.variant}
-          config={config?.config}
-          styles={config?.styles}
-          hasIndicatorIcon={config?.hasIndicatorIcon}
-        />
-      )}
+      {!isEvt &&
+        !child &&
+        (readonly ? (
+          <Attribute
+            key={code}
+            attribute={code}
+            code={config?.parentCode || ask?.targetCode}
+            size={config?.size}
+            mini={config?.mini}
+            parentCode={config?.parentCode || ask?.targetCode}
+            variant={config?.variant}
+            config={config?.config}
+            styles={config?.styles}
+            hasIndicatorIcon={config?.hasIndicatorIcon}
+          />
+        ) : (
+          <Ask
+            key={code}
+            parentCode={config?.parentCode || ask?.parentCode}
+            questionCode={ask?.code}
+            config={config?.config}
+            noLabel={undefined}
```

Comment:
You don't need to pass the value as undefined, just don't pass the props at all, it will default to undefined.